README.md

```diff
@@ -562,7 +580,14 @@ WebHttpClient.builder()
         .build()    
 ```
 
-### Create http GET request object
+### Create an HTTP request
+The requests available are: 
```

Comment:
The target endpoint's http method is defined in the request object. Thus, we should say the request supports the following http methods as suggested bellow. Up to you if you want to include the explanation on the method is defined in the request object.

```suggestion
The request supports the following http methods: 
```